CODE_OF_CONDUCT.rst

```diff
@@ -34,8 +34,6 @@ Actions that will not be tolerated include, but are not limited to:
 - Discrimination in Any Form: Discriminating behaviour or conduct relating to, but not limited to, any of the
   aforementioned characteristics.
 
-|
```

Comment:
What's the reason for removing the additional vertical spacing? Having that only helps readability.

Reply:
Natural writing and paragraphs don't have double-space separation. It dosen't help readability it just makes it look like a mistake and irregular since other repos don't have it.

Reply:
I disagree, I think the added vertical space before a new section makes it easier to read as it's less "crowded".

Reply:
While I appreciate the dissent it's not really an opinion thing. Double spaces like that are not used commonly between sections in writing or github in general. You can pick pretty much any repo: [Roslyn](https://github.com/dotnet/roslyn), [Rust](https://github.com/rust-lang/rust), [Tensorflow](https://github.com/NVIDIA/tensorflow), [pip](https://github.com/pypa/pip). Very few do it. It wouldn't be how I would write it right now if I decided to use subsections either.

You come to our repo and it immediately looks off because it's not consistent with how the rest of the site works and how people read in general. At best anyone visiting thinks we just made an obvious mistake and haven't fixed it, or considered our stylistic choices more important than consistency with the rest of the platform. At worst people will judge it as a bad idea and read our project and code in a dimmer view or simply not bother reading it at all because it looks weird to their eyes. 

Being consistent with the rest of the site, how people have come to know things to be formatted, and using normal spacing avoid all these situations with zero downsides.

Right now the first impact anyone has when seeing our repository is not, 'oh lets see what a quantum compiler actually is!', it's 'why is the spacing weird'.